### DISCUSSION TITLE
Global File Search in SFTPGO

### DISCUSSION THREAD

**Discussion by CEckelberry** [drakkan/sftpgo#1736] [Q&A]
All,

My company is thinking about adopting SFTPGO as our solution to manage GCS Buckets, NFS Filestore Instances, and AWS S3 buckets. Is there a way to enable global file level search for users? I can only seem to search in a folder once I get into the folder with the files themselves instead of being able to search from the home screen. Pictures attached. 

![image](https://github.com/user-attachments/assets/38c58472-a445-4eb4-9df6-d01aa2ccc53b)

![image](https://github.com/user-attachments/assets/f555def1-114c-4bd1-9a87-7487c3297254)

**Comment by vjp23** [comment]
Has there every been any other discussion around this? It's maddening.
